flink-connectors/flink-connector-elasticsearch7/pom.xml

```diff
@@ -100,6 +100,13 @@ under the License.
 
 		<!-- Tests -->
 
+		<dependency>
+			<groupId>org.apache.flink</groupId>
+			<artifactId>flink-connector-base</artifactId>
```

Comment:
add a comment that this is required by elasticsearch-base. (as is it could probably be detected as unused)